homeassistant/components/homekit/__init__.py

```diff
@@ -0,0 +1,114 @@
+"""Support for Apple Homekit.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/homekit/
+"""
+import asyncio
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import (
+    ATTR_FRIENDLY_NAME, ATTR_SUPPORTED_FEATURES, CONF_PATH, CONF_PORT,
+    EVENT_HOMEASSISTANT_START, EVENT_HOMEASSISTANT_STOP)
+from homeassistant.util import get_local_ip
+from homeassistant.util.decorator import Registry
+
+TYPES = Registry()
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'homekit'
+REQUIREMENTS = ['HAP-python==1.1.2']
+
+BRIDGE_NAME = 'Home Assistant'
+CONF_PIN_CODE = 'pincode'
+
+CONFIG_SCHEMA = vol.Schema({
+    DOMAIN: vol.All({
+        vol.Optional(CONF_PORT, default=51826): vol.Coerce(int),
+        vol.Optional(CONF_PIN_CODE, default='123-45-678'): cv.string,
+        vol.Optional(CONF_PATH, default='accessory.state'): cv.string,
```

Comment:
This should not be configurable. There should just be 1 config. Also, please add a `.` in front of the name as it's not supposed to be edited by users.

Reply:
> Also, please add a `.` in front of the name as it's not supposed to be edited by users.

Do you mean `_BRIDGE_NAME`?

Which options would you remove. It is recommended to use something else then `123-45-678` as pin code. Unless we define something and write it all over the docs, we should keep it. `Port` ok, quite unlikely that it will be blocked be other services. `Path`?

Reply:
Filenames starting with a `.` are hidden by default. So make it `.accessory.state`